components/script/dom/webidls/DOMMatrixReadOnly.webidl

```diff
@@ -81,6 +81,6 @@ interface DOMMatrixReadOnly {
     DOMPoint            transformPoint(optional DOMPointInit point = {});
     Float32Array        toFloat32Array();
     Float64Array        toFloat64Array();
-//    [Exposed=Window] stringifier;
+    [Exposed=Window, Throws] stringifier;
```

Comment:
Note that the `Throws` attribute is not present in https://drafts.fxtf.org/geometry-1/#DOMMatrix, but the stringifier is definitely fallible (Step 1.)